src/java.base/share/classes/java/math/MutableBigInteger.java

```diff
@@ -43,6 +43,7 @@
  * @see     BigInteger
  * @author  Michael McCloskey
  * @author  Timothy Buktu
+ * @author  Fabio Romano
```

Comment:
In recent years we no longer add the @author tag. In fact, at some point we plan to remove them from the codebase.
Authorship is part of the git commits.